packages/atomic/src/components/atomic-breadbox/atomic-breadbox.tsx

```diff
@@ -58,12 +62,67 @@ export class AtomicBreadbox implements InitializableComponent {
   @State() public error!: Error;
   @State() private isCollapsed = true;
 
-  // TODO: KIT-924 Automatically adapt the number of element of the breadbox
-  private collapseThreshold = 5;
-
   public initialize() {
     this.breadcrumbManager = buildBreadcrumbManager(this.bindings.engine);
     this.facetManager = buildFacetManager(this.bindings.engine);
+    this.resizeObserver = new ResizeObserver(() => this.adaptBreadcrumbs());
+    this.resizeObserver.observe(this.host.parentElement!);
```

Comment:
Inspired from the field values list, you'll see that I hide the elements with DOM manipulation (setting style display none & updating show more text) after:
- a resize of the list
- a re-render of the component

Another approach could have been to re-render multiple times instead, but it seamed to be less performant and more complex to manage.

Reply:
That said, I'll explore a bit further using only JSX + internal state

Reply:
Would a combination of the two approaches be the simplest?
- Using `resizeObserver` to calculate which breadcrumbs to display,
- Setting this array in state e.g. `visibleBreadcrumbs`,
- Managing the render with JSX to display only the `visibleBreadcrumbs`, the correct show more count, aria-label

Reply:
If you end up using Sami's solution, please tag me and I'll create a JIRA to update the fields list component.